File: sdk/unity/sdk/Assets/KingstonDynamics/DragonSong/Protocol/Utils/Encodeinator.cs
using System;

namespace KingstonDynamics.DragonSong.Protocol.Utils
{
    public class Encodeinator
    {
        public static byte[] Encodeinate(byte[] bytes)
        {
            return Byteinator.StringToBytes(Convert.ToBase64String(bytes));
        }

        public static byte[] Decodeinate(byte[] bytes)
        {
            return Convert.FromBase64String(Byteinator.BytesToString(bytes));
        }
    }
}

File: src/old/DragonSong-API/src/main/java/com/kingstondynamics/dragonsong/api/util/misc/Encodeinator.java
package com.kingstondynamics.dragonsong.api.util.misc;

import java.util.Base64;

public class Encodeinator {
    
    // Blocked Private Constructor
    // The Exception prevents initialization via reflection
    private Encodeinator() {
        throw new UnsupportedOperationException();
    }
    
    public static byte[] encodeinate(byte[] bytes) {
        return Base64.getEncoder().encode(bytes);
    }
    
    public static byte[] decodeinate(byte[] bytes) {
        return Base64.getDecoder().decode(bytes);
    }
}
